ui/app/src/views/home/ImportantDashboards.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box } from '@mui/material';
+import { useImportantDashboardList } from '../../model/dashboard-client';
+import { DashboardList } from '../../components/DashboardList/DashboardList';
+
+interface ImportantDashboardProps {
+  id?: string;
+}
+
+export function ImportantDashboard(props: ImportantDashboardProps) {
```

Comment:
Maybe rename this to `ImportantDashboardList`?

Reply:
I will move it at the same place as RecentDashboardList & DashboardList components